### ISSUE TITLE
Primary Admin account notifications not working in some cases

### ISSUE THREAD

**Issue by eliassal** [Cacti/cacti#4529]
# Describe the bug

I have a fresh new setup, 5 days ago, I have finalized admin user properties with email but still getting 

> SYSTEM WARNING: Primary Admin account notifications disabled! Unable to send administrative Email.

## To Reproduce

1. Go to Log

2.Warning displayed

## Expected behavior

This should disappear otherwise, where can we setup the notification?
## Screenshots

![Cacti-warn](https://user-images.githubusercontent.com/8447864/149982920-c0556e22-4904-4871-a5dd-88d61420e084.jpg)

![Cacti-warn_02](https://user-images.githubusercontent.com/8447864/149982982-e4ed0e4e-1749-4a34-a492-84ff97dfc7d2.jpg)

## Desktop (please complete the following information)

- OS: Fedora 35

- Edge, chrome, Firefox

- Version 1.2.19

**Comment by TheWitness** [comment]
Edit the Admin account and setup it's email address.

**Comment by eliassal** [comment]
It is already done

**Comment by TheWitness** [comment]
Okay, that's odd.  Let me check.

**Comment by TheWitness** [comment]
Okay, looked at the code, and I'm going to call bug.  Updating `lib/functions.php` shortly.  Please test.

**Comment by eliassal** [comment]
Thanks, so what should I do in my environment?

**Comment by TheWitness** [comment]
Did you install from source?  If so, the easy process, ntong to verify the user accounts for ownership and that the paths may be different....

```sh
cd /var/www/html
git clone -b 1.2.x https:/github.com/cacti/cacti.git cacti.new
/bin/cp -rfp cacti.new/* cacti
chown -r apache:apache cacti
```

Then refresh your browser to upgrade.

**Comment by eliassal** [comment]
No, I followed the instructions at
https://www.tecmint.com/install-cacti-network-monitoring-on-rhel-centos-fedora/
and used 

> sudo dnf install cacti

**Comment by TheWitness** [comment]
It's likely going to be a bit more tricky adding @reubenfarrelly as he can add some clarity as to when a new build might be available.  We plan to freeze later today after all the language stuff is finished.

**Comment by TheWitness** [comment]
It's working for me now.  I'm going to close now.  Just wait on 1.2.20 to be released if you don't know how to merge.

**Comment by nani81** [comment]
Good evening,
I found this article because the same thing is happening to me in my cacti.

So I've added the fix in my functions.php file, but I still get the same error.

Versión: cacti 1.2.19
OS: Red Hat
